qc-runner/src/cellSizeDistribution.r

```diff
@@ -82,9 +82,20 @@ task <- function(seurat_obj, config, task_name, sample_id) {
     # 3483 6019 3892 3729 4734 3244
     
     plot2_data <- seq_along(plot1_data)
-    plot2_data <- unname(map2(plot1_data,plot2_data,function(x,y){c("u"=x,"rank"=y)}))
+    plot1_data_transformed <- (log(plot1_data)/log(10))*2200
```

Comment:
I suppose this is a design choice to have the log computed on the pipeline side rather than the UI side (i.e. through a checkbox to activate/dactivate log transformation?

Reply:
Yes, was a design from the engineer side. I think that it is something that we are not going to change, because for the knee plot we use always the log transformed. But, I am happy to control it by a parameter if you think that we need to keep also the non transformed list

Reply:
Ok, I think we can park this discussion for another time, it's ok for now. My thinking was to also be able to control the y axis with a log checkbox going forward. Let's discuss in the bionfo meeting.

Reply:
Juanlu, could you explain a bit more about that normalization? why the 2200 hardcoded value?

Reply:
Agi gave me this formula, I think that It should be explained in the design document or probably Seb has more knowledge here since he developed this filter.

Reply:
The formula was the in the UI, I also have no idea what the 2200 is for,